calculator.js

```diff
@@ -442,7 +442,7 @@ $( document ).ready(function() {
     //   - Change "theme name" to a catchy name (can be your name)
     //   - Change the colors blue and red with colors of your choice.
     //   - Uncomment the call to addOption (remove the `//`)
-    // addOption("theme name", "linear-gradient(blue, red)", "red", "red");
+    addOption("alango name", "linear-gradient(red, green)", "blue", "white");
```

Comment:
Consider removing "name" from theme name if you feel like it! 
```suggestion
    addOption("alango", "linear-gradient(red, green)", "blue", "white");
```